src/hooks/useActiveOrg.tsx

```diff
@@ -0,0 +1,48 @@
+import { getActiveOrgId, setActiveOrgId } from "@propelauth/javascript"
+import { useContext, useEffect, useState } from "react"
+import { AuthContext } from "../AuthContext"
+
+const ORG_SELECTION_LOCAL_STORAGE_KEY = "__last_selected_org"
+
+export function useActiveOrg() {
+    const context = useContext(AuthContext)
+    if (context === undefined) {
+        throw new Error("useActiveOrg must be used within an AuthProvider or RequiredAuthProvider")
+    }
+    const [activeOrgIdState, setActiveOrgIdState] = useState<string | undefined>(getActiveOrgId())
+
+    useEffect(() => {
+        const intervalId = setInterval(() => {
+            const currentCookieValue = getActiveOrgId()
+            if (currentCookieValue !== activeOrgIdState) {
+                setActiveOrgIdState(currentCookieValue)
+            }
+            // TODO: What should this value be?
+        }, 500)
```

Comment:
@andrew-propelauth As you can see, there is a slight delay when changing the active org:

https://github.com/PropelAuth/react/assets/36497022/f51e1e50-d065-442b-878e-e0b31ac9c0a8

I was unsure what timing should be set here, as I don't want to exhaust the hook. WDYT?